editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/HomepagePlanningForm.tsx

```diff
@@ -0,0 +1,123 @@
+import BasicRadio from "@planx/components/shared/Radio/BasicRadio";
+import { useFormik } from "formik";
+import React, { ChangeEvent, useState } from "react";
+import InputDescription from "ui/editor/InputDescription";
+import Input from "ui/shared/Input";
+import InputRow from "ui/shared/InputRow";
+import InputRowLabel from "ui/shared/InputRowLabel";
+
+import { SettingsForm } from "../shared/SettingsForm";
+import { FormProps } from ".";
+export default function HomepagePlanningForm({
+  formikConfig,
+  onSuccess,
+}: FormProps) {
+  const [showPlanningInputs, setShowPlanningInputs] = useState(false);
+
+  const formik = useFormik({
+    ...formikConfig,
+    onSubmit(values, { resetForm }) {
+      onSuccess();
+      resetForm({ values });
+    },
+  });
+
+  const onChangeFn = (type: string, event: ChangeEvent<HTMLInputElement>) =>
+    formik.setFieldValue(type, event.target.value);
+
+  const boolTransform = (event: ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === "yes") {
+      setShowPlanningInputs(true);
+      return true;
+    } else if (event.target.value === "no") {
+      setShowPlanningInputs(false);
+      return false;
+    }
+  };
+
+  return (
+    <SettingsForm
+      legend="Homepage and Planning Portal"
+      formik={formik}
+      description={
+        <InputDescription>
+          A link to your homepage displayed publicly to your users to help
+          navigate your council services and a link to your Planning Portal to
+          connect your planning data with our outputs.
+        </InputDescription>
+      }
+      input={
+        <>
+          <InputRow>
+            <InputRowLabel>
+              Homepage URL
+              <Input
+                name="homepage"
+                onChange={(event) => {
+                  onChangeFn("homepage", event);
+                }}
+              />
+            </InputRowLabel>
+          </InputRow>
+          <InputRow>
+            <InputRowLabel>
+              Do you collect Planning data?
+              <RadioGroup
+                name="isPlanningDataCollected"
+                defaultValue={
+                  formik.values.isPlanningDataCollected === true ? "yes" : "no"
+                }
+                onChange={(event) => {
+                  formik.setFieldValue(
+                    "isPlanningDataCollected",
+                    boolTransform(event),
+                  );
+                }}
+              >
+                <BasicRadio
+                  title="Yes"
+                  variant="compact"
+                  id="yes"
+                  value="yes"
+                  onChange={() => {}}
+                />
+                <BasicRadio
+                  title="No"
+                  variant="compact"
+                  id="no"
+                  value="no"
+                  onChange={() => {}}
+                />
+              </RadioGroup>
+            </InputRowLabel>
+          </InputRow>
+          <InputRow>
```

Comment:
So actually I think we can remove this whole block! When we added this feature, we thought councils may want to have customisable URLs here, but so far this hasn't proved to be the case. 

Here's my recommendation - 
 - We keep these columns on the `team_settings` table, so if we need to customise this in future we easily can
 - We don't show this as a feature to the user, and see if we get any feedback
 - In future, we can easily add these form filed, or remove the db columns, based on feedback

Reply:
@DafyddLlyr so remove this who file?

Reply:
Sorry - should have been clearer! Just the inputs for Planning Portal name and Planning Portal URL. We've found that nobody uses custom values here currently 👍

Reply:
Ah okay! but keep the Boolean field and the homepage?

Reply:
Asking because, as I am thinking, if it's just the Homepage + the Bool field, then what use is the bool field? 


If it is just the Homepage, could I just incorporate that into the Contact Form?

Reply:
Homepage should stay, and could move into contact 👍 

Just taken a closer look at `team_settings.has_planning_data` - this looks like it's actually unused. We've recently moved this across to `team_integrations` (PR here - https://github.com/theopensystemslab/planx-new/pull/2776)

What this means is - 
 - We can remove the the form field for `hasPlanningData` - this is an integration, not a setting, and doesn't need to be handled here
 - We can remove the `team_settings.has_planning_data` column (in another PR)

Reply:
Apologies for not spotting this issue with `has_planning_data` sooner!

Reply:
Sick! All makes sense